### DISCUSSION TITLE
Fehlermeldung bei Passworteingabe: TypeError: Failed to fetch

### DISCUSSION THREAD

**Discussion by wolwes** [tbnobody/OpenDTU#648] [Q&A]
Ich habe in Edge ein Problem mit dem Passwort von Open DTU. Ich habe irgendwann das Standart-Passwort - wie empfohlen - verändert. Ich bin dabei aber von einem Anruf gestört worden, so dass ich das blödsinnigerweise sehr hastig und unkonzentriert vorgenommen habe. Nach kurzer Zeit wusste ich also schon nicht mehr, was ich da eigentlich eingegeben habe...Jetzt hatte ich den Salat und konnte nun keine Äderungen an den Einstellungen mehr vornehmen. Ich weiß, das war ziemlich idiotisch, aber so ist es nun mal.

Irgendwann kam ich auf die Idee, es in einem anderen Browser zu versuchen und siehe da. Ich kam in Google Chrome mit "openDTU42" hinein und konnte auf Werkseinstellung das DTU zurücksetzen. Nun bin Ich ganz glücklich, da ich mich nun um das MQTT und das Display kümmern kann. Aber nur in Chrome! Denn wenn ich in Edge die Open DTU Oberfläche öffne, erhalte ich die im Titel genannte Fehlermeldung, wenn ich das Standard Passwort eingebe. Da ich aber fast nur in Edge unterwegs bin, nervt mich das.

Also, wie und was kann man machen???
tschau
Wolfgang

**Comment by tbnobody** [comment]
Hallo Wolfgang,

wie ist denn der Login-Status in Edge? (Farbe des Buttons oben rechts)

**Reply by wolwes** [reply]
Entschuldigung, ich musste mal kurz weg...
ich hABE gerade mal reingeschaut. Wenn ich mit der Maus drübergehe ist der Button grün. Die Werte werden angezeigt, nur in die Einstellungen komme ich nicht.

**Comment by tbnobody** [comment]
Und wo bekommst du die Fehlermeldung? Bitte um Screenshots

**Comment by wolwes** [comment]
![OpenDTU](https://user-images.githubusercontent.com/16593975/222442872-3b04c04d-80cc-4462-9278-4faeb3004443.png)
ich kenne dieses Forum überhaupt nicht, habe jetzt einfach das Bild hier reingezogen, hoffe es klappt...

**Comment by wolwes** [comment]
Ich muss das hier jetzt leider unterbrechen, weil ich nochmals weg muss. Ab 18 Uhr kann es dann weitergehen.... 
Aber bis dahin, erst einmal vielen Dank für die prompte Antwort und das ernsthafte Kümmern um dieses merkwürdige Verhalten des Browsers Edge.
tschau
Wolfgang

**Comment by tbnobody** [comment]
Ich habs hier gerade auch nochmal mit Edge versucht (Version 110.0.1587.57 (Offizielles Build) (64-Bit)) --> Klappt Problemlos. 

Hast du mal versucht die Seite mit Strg+F5 neu zu laden? Ggf. ist da noch  etwas altes im Cache.

**Comment by wolwes** [comment]
klar, das war das Erste, was ich probiert habe. Ich habe gerade den Desktop-Rechner zum ersten Mal heute angemacht und bin gerade in Edge gegangen, um das Ganze nochmals zu probieren. Es bleibt bei der Fehlermeldung. Das Merkwürdige ist, dass ich jetzt ganz oben darauf hin gewiesen wurde, ich möge doch das Standard-Passwort ändern. Das bedeutet doch aber, dass das "System" davon ausgeht, dass irgendwo "openDTU42" noch eingetragen ist. Aber wenn ich eben dieses eingebe, kommt die Fehlermeldung.

klar, kann ich verstehen, dass alles reibungslos bei Dir funktioniert. War bei mir vor der Rücksetzung auch so...

**Comment by tbnobody** [comment]
Die meldung mit dem standard passwort bekommst du im chrome aber auch oder?

**Comment by wolwes** [comment]
oh, da habe ich noch gar nicht drauf geachtet...Probier ich jetzt.

**Comment by wolwes** [comment]
nein, ich habe sofort versucht, in die Einstellungen zu kommen. Daraufhin musste ich das Paswort eingeben. Habe openDTU42 eingetragen und schon war ich in den Einstellungen. Keine Meldung oder Warnung...

**Comment by wolwes** [comment]
habe gerade das Display angeschlossen und die richtigen Einstellungen vorgenommen. Zeigt jetzt offline -logisch-. Ich bin begeistert.  Jetzt werde ich das Layout im Easyeda ändern und die Platinenbestellung vornehmen...

**Comment by wolwes** [comment]
Beim Wechseln von den Einstellungen auf Live Daten kam die Warnung: "Sie verwenden das Standardpasswort für die Weboberfläche und den Notfall Access Point. Dies ist potenziell unsicher. [Bitte ändern Sie das Passwort.]"
(in Chrome)
